Sources/NetworkService/ParametersEncoder/URLParametersEncoder.swift

```diff
@@ -10,7 +18,14 @@ public struct URLParametersEncoder: ParametersEncoder {
         request.url = urlComponents?.url
     }
 
-    private static func queryItems(from parameters: URLParameters) -> [URLQueryItem] {
+    /// Create a collection of query items from the provided set of URL parameters. Each key-value
+    /// pair of URL parameters corresponds to a query item, where the key is the query item's name
+    /// and the value is the query item's value.
```

Comment:
I think we can remove this sentence because it describes the implementation. 
`Each key-value pair of URL parameters corresponds to a query item, where the key is the query item's name and the value is the query item's value.`
WDYT?